Assets/MRTK/Providers/OpenXR/Scripts/HolographicReprojectionMethod.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.MixedReality.Toolkit.XRSDK.OpenXR
+{
+    /// <summary>
+    /// Enumeration defining how holograms are stabilized during reprojection.
+    /// </summary>
+    public enum HolographicReprojectionMethod
```

Comment:
Do we need to sync this with the non-OpenXR reprojection enum?

Reply:
We can, but since that one is specifically in the WMR folder and they aren't currently used in a fully platform-abstracted way, I chose to create a new enum to fully match OpenXR's support.

The main difference is that OpenXR appears to have kinda collapsed two concepts into one: the "depth vs autoplanar" enum plus https://docs.microsoft.com/en-us/uwp/api/windows.graphics.holographic.holographicreprojectionmode?view=winrt-19041.